images/linux/scripts/installers/git.sh

```diff
@@ -35,3 +35,11 @@ echo "Lastly, document the installed versions"
 DocumentInstalledItem "Git ($(git --version 2>&1 | cut -d ' ' -f 3))"
 # git-lfs/2.6.1 (GitHub; linux amd64; go 1.11.1)
 DocumentInstalledItem "Git Large File Storage (LFS) ($(git-lfs --version 2>&1 | cut -d ' ' -f 1 | cut -d '/' -f 2))"
+
+#Install hub
+snap install hub --classic
```

Comment:
Please update documentation to provide hub version

Reply:
Done: https://github.com/andy-mishechkin/virtual-environments/commit/9eb8d744a7f31c7d839aa8ea6a5fc21ca0147e5c
Builds:
https://github.visualstudio.com/virtual-environments/_build/results?buildId=62031&view=logs&j=011e1ec8-6569-5e69-4f06-baf193d1351e&t=486c0909-f67d-5748-80b3-bb1d531e7b43
https://github.visualstudio.com/virtual-environments/_build/results?buildId=62030&view=logs&j=011e1ec8-6569-5e69-4f06-baf193d1351e&t=486c0909-f67d-5748-80b3-bb1d531e7b43
https://github.visualstudio.com/virtual-environments/_build/results?buildId=62028&view=logs&j=011e1ec8-6569-5e69-4f06-baf193d1351e&t=486c0909-f67d-5748-80b3-bb1d531e7b43
https://github.visualstudio.com/virtual-environments/_build/results?buildId=62029&view=logs&j=011e1ec8-6569-5e69-4f06-baf193d1351e&t=486c0909-f67d-5748-80b3-bb1d531e7b43